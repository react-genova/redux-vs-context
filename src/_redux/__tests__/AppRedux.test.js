import React from 'react';
import AppRedux from '../AppRedux';
import { render } from 'react-testing-library';

describe('redux AppRedux', () => {
    it('renders without crashing with no children', () => {
        const { container } = render(<AppRedux />);
        expect(container.firstChild).toMatchSnapshot();
    });

    it('renders without crashing with children', () => {
        const { container } = render(<AppRedux><div /></AppRedux>);
        expect(container.firstChild).toMatchSnapshot();
    });
});