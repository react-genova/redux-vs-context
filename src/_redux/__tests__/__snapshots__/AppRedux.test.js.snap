// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`redux AppRedux renders without crashing with children 1`] = `<div />`;

exports[`redux AppRedux renders without crashing with no children 1`] = `null`;
